From a8dd293d502ec1c53a2a275cf4f5fb2f89191ce9 Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <B38613@freescale.com>
Date: Thu, 26 Jul 2012 11:09:10 +0800
Subject: [PATCH 034/310] ENGR00218288 GPS:sync gps config for arm2 and
 sabreauto

sync gps config settings with sabresd for the newest version.

Signed-off-by: Jianzheng Zhou <B38613@freescale.com>
---
 .../overlay/frameworks/base/core/res/res/values/config.xml    | 11 +++++++++++
 .../overlay/frameworks/base/core/res/res/values/config.xml    | 10 ++++++++++
 2 files changed, 21 insertions(+)

diff --git a/imx6/arm2/overlay/frameworks/base/core/res/res/values/config.xml b/imx6/arm2/overlay/frameworks/base/core/res/res/values/config.xml
index 7c6e23d..de5a6f0 100755
--- a/imx6/arm2/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/imx6/arm2/overlay/frameworks/base/core/res/res/values/config.xml
@@ -86,4 +86,15 @@
          movement threshold where scrolling should begin. -->
     <dimen name="config_viewConfigurationTouchSlop">24dp</dimen>
 
+    <!-- Component name of the service providing network location support. -->
+    <string name="config_networkLocationProvider">com.google.android.location.NetworkLocationProvider</string>
+
+    <!-- Component name of the service providing geocoder API support. -->
+    <string name="config_geocodeProvider">com.google.android.location.GeocodeProvider</string>
+
+    <!-- add gpsVendor to support some special code from different gps vendor -->
+    <add-resource type="string" name="config_gpsVendor" />
+    <string name="config_gpsVendor">atheros</string>
+
+
 </resources>
diff --git a/sabreauto_6q/overlay/frameworks/base/core/res/res/values/config.xml b/sabreauto_6q/overlay/frameworks/base/core/res/res/values/config.xml
index 7c6e23d..4770cd7 100755
--- a/sabreauto_6q/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/sabreauto_6q/overlay/frameworks/base/core/res/res/values/config.xml
@@ -86,4 +86,14 @@
          movement threshold where scrolling should begin. -->
     <dimen name="config_viewConfigurationTouchSlop">24dp</dimen>
 
+    <!-- Component name of the service providing network location support. -->
+    <string name="config_networkLocationProvider">com.google.android.location.NetworkLocationProvider</string>
+
+    <!-- Component name of the service providing geocoder API support. -->
+    <string name="config_geocodeProvider">com.google.android.location.GeocodeProvider</string>
+
+    <!-- add gpsVendor to support some special code from different gps vendor -->
+    <add-resource type="string" name="config_gpsVendor" />
+    <string name="config_gpsVendor">atheros</string>
+
 </resources>
-- 
1.8.0

